### ISSUE TITLE
How to get spelling suggestions to work with solr 4.9.1 and update documentation?

### ISSUE THREAD

**Issue by sampeka** [django-haystack/django-haystack#1242]
Hi there,

Trying to get spelling suggestions to work with a Solr 4.9. The problem is that the `spelling_suggestion` method on a `SearchQuerySet` always returns None. I've set `INCLUDE_SPELLING` to `True` in the default haystack connections dict, but  I think the documentation must be missing a step for setting up something on the Solr backend?

I've tried the advice shared here, but didn't work:
https://github.com/django-haystack/django-haystack/issues/844

Also found some advice here, using solr 4.3:
http://stackoverflow.com/questions/17060612/spelling-suggestions-showing-none-haystack

And for reference here's a couple of really old SO questions, too old to help:
http://stackoverflow.com/questions/4594782/spelling-suggestions-with-django-haystack-and-solr
http://stackoverflow.com/questions/17060612/spelling-suggestions-showing-none-haystack

It would be really great if somebody could provide an up-to-date guide with getting search suggestions working with Solr, I'd happy to write it up neatly and submit a pull if anyone could help to get it to work?

**Comment by sampeka** [comment]
Closing this issue, don't think more open issues are helpful here at the moment

**Comment by jpatel3** [comment]
@sampeka Were you able to figure it out? I am running into same issue. I have followed the part of the documentation which allow to get spelling suggestion work, though some of the setting suggested in documentation doesn't seem to be valid any longer as those syntax has been changed in solr. 

eg. [documentation](http://django-haystack.readthedocs.io/en/v2.4.1/installing_search_engines.html) suggest to change following -
`<requestHandler name="standard" class="solr.StandardRequestHandler" default="true" />`

But the actual `solrconfig.xml` has no longer that line. 

Also, trying to run query on `/spell` directly on solr query interface does return the suggestions, but using the haystack api, it doesn't return anything. 

If I am able to make it work, will be happy to update documentation.
